pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/JavaInstanceRunnable.java

```diff
@@ -262,10 +268,12 @@ public void run() {
                 }
                 long processAt = System.currentTimeMillis();
                 stats.incrementProcessed(processAt);
+                addLogTopicHandler();
```

Comment:
this seems to be done at per message?

Can we configure the log handler at the beginning of the instance and remove it at the end?

Reply:
We need to send only the log messages inside the function to the log topic. Other logging messages inside the java instance or runtime should not be sent. Thats why we need to do this on a per message basis.

Reply:
okay... I am not sure if there is any performance penalty here. but okay to me for now.